setup.py

```diff
@@ -60,7 +60,8 @@
     print('Adding extra setuptools args')
     extra_setuptools_args = dict(
         zip_safe=False,  # the package can run out of an .egg file
-        include_package_data=True,
+        include_package_data=False,
```

Comment:
Actually, disregard this. I just looked through your commit history on your fork and answered my own question.